frontend/src/views/Results.tsx

```diff
@@ -356,6 +356,7 @@ function GameResultsPage() {
           inviteContent={inviteContent()}
           loading={loading}
           isCurrentPlayer={players[1]?.user.userId === currentUser?.userId}
+          numProblems={game?.problems.length || 1}
```

Comment:
Why is the alternative `1`? Is that just a placeholder because something has to be there when `game?.problems.length` is `undefined`?

Reply:
Yeah, I don't know when such a case would be, but VSCode read that as a error without the `|| 1`.